src/frontend/js/widgets/Dashboard/components/DashboardSidebar/_styles.scss

```
@@ -0,0 +1,87 @@
+.dashboard-sidebar {
+  flex-shrink: 0;
+  width: 300px;
+  background-color: r-theme-val(dashboard-sidebar, background-color);
+  border-radius: $border-radius;
+  box-shadow: r-theme-val(dashboard-sidebar, base-shadow);
+  color: r-theme-val(dashboard-sidebar, base-color);
+  padding-bottom: rem-calc(27px);
+
+  @include media-breakpoint-down(lg) {
+    width: auto;
+  }
+
+  &__container {
+    &__header {
+      padding: rem-calc(64px) rem-calc(16px) rem-calc(50px) rem-calc(16px);
+      position: relative;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+
+      &__avatar {
+        position: absolute;
+        top: calc($avatar-size / -2);
+      }
+
+      h3 {
+        font-weight: bold;
+        text-align: center;
+        margin-bottom: rem-calc(35px);
```

```suggestion
        margin-bottom: rem-calc(12px);
```

By testing, I found the space between header title and catchphrase too large. I take a look on mockup, then I think we should use this page as reference about that. From my point of view, pages with large padding have mistake.

https://www.sketch.com/s/c0aaee1b-8058-4471-b744-771ed7356622/a/yg7QAvK#Inspect